product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/index.mdx

```diff
@@ -5,42 +5,37 @@ redirects:
+   | Cluster type | Cluster Info Tab | You can't switch from a single node cluster or a high availability cluster to an extreme high availability cluster or vice versa. |
+   | Number of replicas (for a high availability cluster) | Cluster Info tab | \-                                                                                                                                                                                                                                                                                 |
+   | Cluster name and password                            | Cluster Settings | \-                                                                                                                                                                                                                                                                                 |
+   | Instance type                                        | Cluster Settings |Changing the instance type can incur higher cloud infrastructure charges.                                            |
+   | Volume type                                          | Cluster Settings | You cannot switch between the io2 and io2 Block Express volume types in an AWS cluster.                                                                                                                                                                                          |
+   | Volume properties                                    | Cluster Settings | It can take up to six hours to tune IOPS or resize the disks of your cluster because AWS requires a cool down period after volume modifications as explained in the [Limitations](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/modify-volume-requirements.html) section. The volume properties are disabled and cannot be modified while this is in progress. |
+   | Networking type (public or private)                  | Cluster Settings | If you are using Azure and previously setup a private link and want to  change to a public network, you must remove the private link resources  before making the change.                                                                                                        |
+   | Database configuration parameters                    | DB Configuration | If you are using faraway replicas, there are certain parameters that need to modified in the replica when increased in the replica's parent cluster. See [Modify a faraway replica](/biganimal/latest/using_cluster/managing_replicas/#modify-a-faraway-replica/).                                                                                                                                                                                                                                                                                |
```

Comment:
should we add here that the user can't set any GUCs on replicas apart from the 5 GUCs listed on the Managing Replicas page?

Reply:
done